manifests/params.pp

```diff
@@ -7,8 +7,20 @@
   $package_version = 'latest'
   $srvadmin = true
   $srvadmin_package = 'srvadmin-all'
+  $srvadmin_packages_java = ['srvadmin-all', 'srvadmin-webserver', 'srvadmin-tomcat', 'srvadmin-jre', ]
```

Comment:
have you tested removing `srvadmin-all` doesn't remove all the dependencies?

Reply:
Yes I have. Removing `srvadmin-all` only removes that package, installing `srvadmin-all` installs all the other packages PLUS the java related packages.